user_guide_src/source/changelogs/v4.2.0.rst

```diff
@@ -0,0 +1,10 @@
+Version 4.2.0
+=============
+
+Release Date:
+
+**4.2.0 release of CodeIgniter4**
+
+Changes:
+
+- ``Validation::setRule`` signature changed and now allows any type for ``$rules``; see PR #3910
```

Comment:
Although technically true because of PHP 7's type limitations, let's make this explicit to the actual types:
```suggestion
- ``Validation::setRule`` signature changed and now allows string or array types for ``$rules``; see PR #3910
```